src/main/java/core/basesyntax/UserInterface.java

```diff
@@ -0,0 +1,99 @@
+package core.basesyntax;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.util.Scanner;
+
+public class UserInterface {
+    Scanner scanner = new Scanner(System.in);
+
+    public void userInput() {
+        String inputStr;
+        inputStr = scanner.nextLine();
+        List<String> splitInputStr = new ArrayList<>(Arrays.asList(inputStr.split(" ")));
+        FileEditor fileEditor = new FileEditor();
+        while (!splitInputStr.get(0).equals("exit")) {
+            switch (splitInputStr.get(0)) {
+                case "create": {
+                    fileEditor.createFile(splitInputStr.get(1), splitInputStr.size() == 3
+                            ? splitInputStr.get(2) : " ");
+                    break;
+                }
+                case "read": {
+                    fileEditor.readFile(splitInputStr.get(1), splitInputStr.size() == 3
+                            ? splitInputStr.get(2) : " ");
+                    break;
+                }
+                case "info": {
+                    fileEditor.getInfo(splitInputStr.get(1), splitInputStr.size() == 3
+                            ? splitInputStr.get(2) : " ");
+                    break;
+                }
+                case "help": {
+                    if (splitInputStr.size() == 1) {
+                        getHelp();
+                    } else {
+                        getCommandHelp(splitInputStr.get(1));
+                    }
+                    break;
```

Comment:
Optional: You can create map with key = command and value = description and depends on splitInputStr.size() it will print either COMMAND_INFO.get(command); or COMMAND_INFO.values(). this way you can make this method much shorter and get rid of showHelp() method